zerver/lib/test_helpers.py

```diff
@@ -208,28 +226,37 @@ def __init__(self, post_data, user_profile, assert_callback=None):
 
 class AuthedTestCase(TestCase):
     # Helper because self.client.patch annoying requires you to urlencode
+
     def client_patch(self, url, info={}, **kwargs):
+        # type: (str, Any, Dict[str, Any]) -> HttpResponse
```

Comment:
I think this should be `(str, Dict[str, Any], **Any) -> HttpResponse` (same with the next few)